packages/web-react/src/components/TextField/stories/TextFieldType.tsx

```diff
@@ -5,9 +5,13 @@ import TextField from '../TextField';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const Story = (props: unknown) => (
   <div style={{ display: 'flex', flexWrap: 'wrap', gap: '1.5rem 1rem' }}>
-    <TextField id="textfield-text" label="Type text" name="textfield-text" type="text" />
     <TextField id="textfield-email" label="Type email" name="textfield-email" type="email" />
+    <TextField id="textfield-number" label="Type number" name="textfield-number" type="number" />
     <TextField id="textfield-password" label="Type password" name="textfield-password" type="password" />
+    <TextField id="textfield-search" label="Type search" name="textfield-search" type="search" />
```

Comment:
We might wanna check with designers the number arrows and search clear in the future. Now it works :)

Reply:
I think that may have been the reason why these types were not listed here (TODO: drop a comment next time). But:

1. we suggest using them in the `web` package,
2. we need the `number` type in Jobs 🤷🏻.